src/main/webapp/my/projects.jsp

```diff
@@ -5,8 +5,7 @@
 --%>
 <%@page contentType="text/html" pageEncoding="UTF-8"%>
 <%@ include file="/WEB-INF/includes/include.jsp"%>
-<!DOCTYPE html!>
-<html>
+<!DOCTYPE html>
```

Comment:
@MatzElectronics Are the updates to fix whatever breaks when the <!DOCTYPE html> tag is used in this release?

Reply:
Should be, if they aren't, the editor would appear to be blank (no blocks would be visible).  The main fix is in the CSS file for the editor (style-editor.css I think)